docs/docs/sass.md

```diff
@@ -1,30 +1,30 @@
 ---
-title: Using Sass in Gatsby
+title: Uso de Sass en Gatsby
 ---
 
-[Sass](https://sass-lang.com) is an extension of CSS, adding nested rules, variables, mixins, selector inheritance, and more. In Gatsby, Sass code can be translated to well-formatted, standard CSS using a plugin.
+[Sass](https://sass-lang.com) es una extensión de CSS, que agrega reglas anidadas, variables, mixins, herencia de selectores y más. En Gatsby, el código Sass se puede traducir a CSS estándar, en el formato correcto, mediante un complemento.
```

Comment:
I translated `well-formatted, standard CSS` into `CSS estándar, en el formato correcto`, rather than `CSS estándar, bien formateado` since it sounds more natural to me.